MetriCam2_SDK.sln

```diff
@@ -299,6 +301,18 @@ Global
 		{41D6E896-FB86-4BA1-BC99-E2FE4AA89E93}.Release|x64.Build.0 = Release|Any CPU
 		{41D6E896-FB86-4BA1-BC99-E2FE4AA89E93}.Release|x86.ActiveCfg = Release|Any CPU
 		{41D6E896-FB86-4BA1-BC99-E2FE4AA89E93}.Release|x86.Build.0 = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|x64.Build.0 = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Debug|x86.Build.0 = Debug|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|Any CPU.Build.0 = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|x64.ActiveCfg = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|x64.Build.0 = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|x86.ActiveCfg = Release|Any CPU
+		{4A36576E-E47F-4993-AD94-D09B989B4CE4}.Release|x86.Build.0 = Release|Any CPU
```

Comment:
Any reason why some projects default to x64 and some to Any CPU? Shouldn't x64 be preferred especially for new projects?